bazel/osx/crosstool/CROSSTOOL.tpl

```diff
@@ -1270,8 +1275,7 @@ toolchain {
     config_name: "c++-link-interface-dynamic-library"
     action_name: "c++-link-interface-dynamic-library"
     tool {
-      tool_path: "wrapped_clang"
-      execution_requirement: "requires-darwin"
+      tool_path: "DUMMY_TOOL"
```

Comment:
I'm a bit confused by the introduction of `DUMMY_TOOL` here - it doesn't appear to be in `crosstool.tpl.diff` or in the upstream https://github.com/bazelbuild/bazel/blob/master/tools/osx/crosstool/CROSSTOOL.tpl.